benchmark-install-against-revisions.bash

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -o errexit -o pipefail -o nounset
+cd "$(dirname "$0")"
+cargo run --bin=benchmark-install-against-revisions -- "$@"
```

Comment:
Can you add this to `.cargo/config.toml` and `justfile`. Later you can remove this file. I think we should avoid having shell scripts, since we already have Justfile in this project.